### ISSUE TITLE
It needs spaces before and after to work inside HTML

### ISSUE THREAD

**Issue by ppaul** [ali-saleem/anchorme.js#5]
In my project, user writes text in a box, but behind the scenes, <p> tags are inserted (it applies to <div> and so on). I've noticed that anchorme doesn't do the job with this text. Here are some console tests which say everything:

<pre>
anchorme.js('www.google.com')
"<a href='http://www.google.com'>www.google.com</a>"
anchorme.js('<p>www.google.com</p>')
"<p>www.google.com</p>"
anchorme.js('<p> www.google.com </p>')
"<p> <a href='http://www.google.com'>www.google.com</a> </p>"
anchorme.js('<div>google.com</div>')
"<div>google.com</div>"
anchorme.js('<div> google.com </div>')
"<div> <a href='http://google.com'>google.com</a> </div>"
</pre>

**Comment by alexcorvi** [comment]
For some reason I can't reproduce this issue.
I've copied and pasted this: `anchorme.js('www.google.com')` and it's working properly.
For the other ones, my console is returning errors say that there's `Unterminated string constant`

**Comment by ChrisOelmueller** [comment]
Try including HTML tags in the string such as `<p>https://github.com/ali-saleem/anchorme.js</p>` or the rather innocent-looking `&nbsp;https://github.com/ali-saleem/anchorme.js`. What I'm seeing with 0.4.1 is that these links will not be parsed on the anchorme demo page.

**Comment by ChrisOelmueller** [comment]
@ali-saleem Is there currently any way I can convince anchorme to recognize a link such as `<p>https://github.com/ali-saleem/anchorme.js</p>`?

**Comment by kellertobias** [comment]
The error occures, when immediately after the link, a tag is closes:

`anchorme.js("<p>Hallo you!</p><p>see my link: http://www.google.de</p>","")`

if after the link is a space, it works: `anchorme.js("<p>Hallo you!</p><p>see my link: http://www.google.de </p>","")`

**Comment by kellertobias** [comment]
Seems that a fix would be: inside of anchorme.js:

```
    str = str.split("\n").join(" \n ");
    str = str.split(" (").join(" ( ");
    str = str.split(")").join(" )");
    str = str.split("<").join(" <"); //Line Added
    var order1 = anchorme.order(str," ",addattrs);
    var order2 = anchorme.order(order1,"\n",addattrs);
    var order3 = anchorme.order(order2,"<",addattrs); //Line Added
    order2 = order2.split("( ").join("(");
    order2 = order2.split(" )").join(")");
    order3 = order3.split(" <").join("<"); //Line Added
    return order3;
```

**Comment by alexcorvi** [comment]
Issue resolved in 0.6.0 please check.
